sql/pr/top-authors.sql

```diff
@@ -0,0 +1,25 @@
+-- Top authors per year
+WITH mergers_per_year AS (
+    SELECT
+    year(c.commit_time AT TIME ZONE 'UTC') AS year,
+    ai.name || ' <' || ai.email || '>' AS identifier,
+    count(*) AS commits_count
+FROM git.default.commits c
+    JOIN memory.default.idents ai ON ai.email = c.author_email OR CONTAINS(ai.extra_emails, c.author_email)
+    JOIN memory.default.idents ci ON ci.email = c.committer_email OR CONTAINS(ci.extra_emails, c.committer_email)
+WHERE ai.email != ci.email
```

Comment:
Remove this condition, I don't think we should exclude maintainers that merge their own commits.